nx584/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## 1.0.2
```

Comment:
Please fold this into 1.0.1. 1.0.1 was never merged, hence for the users it does not exist.